contracts/AllDay.cdc

```diff
@@ -131,10 +131,13 @@ pub contract AllDay: NonFungibleToken {
         // initializer
         //
         init (id: UInt64) {
-            let series = &AllDay.seriesByID[id] as! &AllDay.Series
-            self.id = series.id
-            self.name = series.name
-            self.active = series.active
+            if let series = &AllDay.seriesByID[id] as &AllDay.Series? {
```

Comment:
Talked to Bastian about best practise syntax for this. Slack thread here for context: https://axiomzen.slack.com/archives/CTGAW0TM2/p1653430599342029